base_rest/models/rest_service_registration.py

```diff
@@ -240,6 +242,13 @@ def _init_global_registry(self):
         _rest_services_databases[self.env.cr.dbname] = services_registry
         return services_registry
 
+    def _register_rest_route(self, route_path):
```

Comment:
@simahawk Be careful, it's not the indent of base_rest to provide a mechanism to allow third party addons to register their own dedicated routes. This kind of approach put a too heavy dependency between addons and technical aspect of this library. 'controller less' route is on the roadmap  next to:
* the entry point concept,
* python typing for the definition of restmethod (as in fastapi),
* refactoring of the openapi documentation generation to be generated at application or entry point level
* add support for odoo pydantic (https://github.com/OCA/rest-framework/pull/218/files -> could replace base_jsonify, cerberus, ....)
* a new abstraction layer on top of the actual components

I'm ok to merge this but this api could be removed into the next evolutions.

Reply:
> I don't mind if it gets refactored. Today I need this because of the patch on `get_request` to select the right request class. Do you think that patch is going to stay anyway?

No motivation to remove-it... therefore it should stay.